### PR TITLE
Remove docs on building dojo custom widgets

### PR DESCRIPTION
**Pull request by mlaponder**
As building pluggable widgets is preferred over building dojo custom widgets, we removed the docs for mx9 on building dojo custom widgets to encourage people to build pluggable widgets over dojo custom widgets.

### CODE REVIEW

**Comment by ConnorLand** [comment]
We are going to merge this mx9 branch once the beta release is approved. Therefore, in a few days I will ask you to re-target your changes against the new Mx9 documentation. I'll ping you when this is possible @mlaponder . Thank you.

**Comment by ConnorLand** [comment]
Based on my conversation with Marco, Mendix 9 marks a time when we are trying to drive customers away from dojo widgets. We actively don't want people to build new ones. Instead, we'd have them build pluggable widgets instead. So I am going to take these Mx9 docs down.

**Comment by ConnorLand** [comment]
I'm going to review/merge this tomorrow.

**Review by ConnorLand — APPROVED** [review]

**Review by ConnorLand — CHANGES_REQUESTED** [review]
Please implement the changes we discussed on slack https://mendix.slack.com/archives/DFYP98R88/p1610962456015000

File: content/apidocs-mxsdk/apidocs/pluggable-widgets-8.md [hunk 1]
```diff
@@ -0,0 +1,285 @@
+---
+title: "Pluggable Widgets API"
```
Comment (ConnorLand) [inline content/apidocs-mxsdk/apidocs/pluggable-widgets-8.md:2]: Based on your confined changes described over slack, are you still going to merge this doc? I don't see how it fits.

File: content/apidocs-mxsdk/apidocs/pluggable-widgets-8.md [hunk 2]
```diff
@@ -0,0 +1,285 @@
...
+category: "API Documentation"
```
Comment (ConnorLand) [inline content/apidocs-mxsdk/apidocs/pluggable-widgets-8.md:3]: The file title of this doc suggests its only for Mendix 8, but the doc title doesn't reflect that.

File: content/apidocs-mxsdk/apidocs/pluggable-widgets-8.md (hunk 1 above)
Comment (mlaponder) [reply content/apidocs-mxsdk/apidocs/pluggable-widgets-8.md:2]: No I am not, this page will no longer be there, I expect to push my changes later today

Comment (ConnorLand) [reply content/apidocs-mxsdk/apidocs/pluggable-widgets-8.md:2]: Ok sounds good.

**Review by ConnorLand — APPROVED** [review]

**Pull request merged**